### PR TITLE
add essential aliases

### PR DESCRIPTION
**Pull request by starsareintherose**
add R and lua

### CODE REVIEW

**Comment by lilydjwg** [comment]
I suppose all the three numbers from R's version are significant as it provides various names with three numbers?

**Comment by starsareintherose** [comment]
So add the version number as alias.

**Comment by starsareintherose** [comment]
fixed https://github.com/archlinuxcn/lilac/pull/187/commits/ee1938a39170dc09f88d2ff981aa2efba1614653

File: lilac2/aliases.yaml [hunk 1]
```diff
@@ -13,6 +13,16 @@ perl:
   alpm: perl
   from_pattern: ^(\d+\.\d+)\..*
   to_pattern: \1
+r:
+  source: alpm
+  alpm: r
+  from_pattern: '^(\d+\.\d+\.\d+)\..*'
```
Comment (lilydjwg) [inline lilac2/aliases.yaml:19]:
```suggestion
  from_pattern: '^(\d+\.\d+\.\d+).*'
```

**Review by lilydjwg — CHANGES_REQUESTED** [review]

**Pull request merged**